onnxruntime/test/contrib_ops/range_test.cc

```diff
@@ -47,6 +47,20 @@ TEST(RangeTest, PositiveInt32Delta_1) {
   test.Run();
 }
 
+TEST(RangeTest, PositiveInt32Delta_2) {
+  OpTester test("Range", 1, onnxruntime::kMSDomain);
+  std::vector<int32_t> start = {1};
+  std::vector<int32_t> limit = {2};
+  std::vector<int32_t> delta = {2};
+  std::vector<int32_t> expected_output = {1};
+
+  test.AddInput<int32_t>("start", {1}, start);
```

Comment:
If start is 1 and limit is 1, isn't the expected output empty?

> np.arange(3.0, 3.0, 5)
array([], dtype=float64)
tf.range also returns []

Reply:
Isn't start 1 and limit 2 here ? This PR addresses the exact case you mention (as stated in the linked issue) and the test case covering that is "SameStartAndLimit"

Reply:
Sorry - was looking at the shape not the data in the AddInput lines.